bitwarden_license/src/Commercial.Infrastructure.EntityFramework/SecretsManager/Repositories/SecretRepository.cs

```diff
@@ -289,6 +295,8 @@ public async Task<(bool Read, bool Write)> AccessToSecretAsync(Guid id, Guid use
         };
 
         var policy = await query.FirstOrDefaultAsync();
+        if (policy == null)
+            return (false, false);
 
         return (policy.Read, policy.Write);
```

Comment:
```suggestion

return policy == null ? (false, false) : (policy.Read, policy.Write);
```

Would you be good with this? 
It is more in line with what we have in the project repository.

Reply:
Yep -- all about following norms, even this fancy one-liner 😜.